client/src/components/ClickToCopy/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+class ClickToCopy extends React.Component {
+  constructor (props) {
+    super(props);
+    this.copyToClipboard = this.copyToClipboard.bind(this);
+  }
+  copyToClipboard (event) {
+    const elementToCopy = event.target.id;
+    const element = document.getElementById(elementToCopy);
```

Comment:
could this just be `const element = event.target;` ?

Reply:
I thought no, but the answer is yes :)